### PR TITLE
Flux profile 2.0

### PR DESCRIPTION
**Pull request by FranklinTrevor**
This pull request lets users selects different flux profiles for TACOCAT
-dictionary was added to LoadedTACO/src/Flux_Profiles to store different flux profiles
-An option was added into TACOCAT read in file for users to select different flux profiles
-TACOCAT was changed to use flux profiles dictionaries

Each flux profile was changed in the read in file and tested to see if TACOCAT outputs the correct flux profile.

### CODE REVIEW

**Comment by lcarasik** [comment]
Can you update this pull request with the other one's merge?

File: tests/test_fluxIntegration.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 import os
 import sys
 sys.path.insert(0,'..') #This adds the ability to pull Flux profiles from the main folder
-import TACOCAT
+from TACOCAT import Tbulk
 
-Test = TACOCAT.test(2,2)
+x = Tbulk
```
Comment (lcarasik) [inline tests/test_fluxIntegration.py]: Can you explain this unit test?

Does it test each profile?

Comment (FranklinTrevor) [reply tests/test_fluxIntegration.py]: This test isn't needed for this pull request it was left over from when I started creating an integration test for flux but realized I needed to do a pull request for updating TACOCAT to be able to use user selected flux profiles. I deleted the file from this pull request to eliminate further confusion.

**Review by lcarasik — APPROVED** [review]
The pull request is atomizitic enough to have small but positive impacts on the code. This is sufficient for moving forward. Approved.

**Pull request merged**